contracts/core/collateral-locker/v1/interfaces/ICollateralLocker.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.6.11;
 
-import "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import { IERC20 } from "../../../../../lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
```

Comment:
why we need relative path for the openzeppelin ? we can keep openzeppelin stuff as it is.

Reply:
No, because when this is imported by another project that may be using a different version of openzeppelin in its root-level libs, this relative git submodule of openzeppelin will still be used. Effectively, this ensures the relative versions of al contracts are used, and no overriding is possible. Eventually, I would even opt to get rid of any "openzeppelin" in the libs, and have it all as "duplicate" and "custom" contract code, of only the exact contracts and functionality needed, keeping the bytecode the same.